include/extractor/extractor.hpp

```diff
@@ -56,14 +56,18 @@ class Extractor
   private:
     ExtractorConfig config;
 
+    std::vector<TurnRestriction> ParseOSMData(ScriptingEnvironment &scripting_environment,
+                                              const unsigned number_of_threads);
```

Comment:
I think we can pull out the number of threads from the type's extractor config, no need to pass it in here

https://github.com/Project-OSRM/osrm-backend/blob/e568b600f0c4ad5a2cd9b8f97868ef9b1b7fc6b2/include/extractor/extractor_config.hpp#L106